linkml-schema/kbmodel.yaml

```diff
@@ -96,34 +78,390 @@ classes:
         inlined_as_list: true
         range: genome assembly
 
+  process:
+    description: >-
+      A process that takes input entities and produces output entities.
+    slots:
+      - input entity
+      - output entity
+      - process id 
+
+
+  brain extraction:
```

Comment:
we may want to create a framework for types of processes. for example these initial processes involve dealing with tissues, others may be more digital, etc.,.